src/app/page/Home/VolunteerHome.jsx

```diff
@@ -101,6 +102,17 @@ export default function VolunteerHome({ currentUser }) {
     updateInProgressMissions(inProgressMissions.filter((m) => m.uid !== missionUid));
   };
 
+  const handleUpdatedMissions = () => {
```

Comment:
You may actually just be able to avoid all of this by using [useFirestoreConnect](https://react-redux-firebase.com/docs/firestore#useFirestoreConnect)
Which will auto sync all your data like we do in the org dashboard https://github.com/factn/resilience-app/blob/7bfc454f154d464a2d6e6162b0a4b02f2f29894d/src/app/dashboard/MissionsControl.jsx#L57

Reply:
Ok, have updated to useFirestoreConnect .  Let me know what you think.